File: cod_python/tabelaHash.py
# votou = dic() # equivalente abaixo
votou = {}

def verificar_eleitor(nome):
	if votou.get(nome):
		return "Ja votou"
	else:
		votou[nome] = True
		return "Deixe votar"


print(verificar_eleitor("Nil"))
print(verificar_eleitor("Joao"))
print(verificar_eleitor("Nil"))

File: cod_javascript/tabelaHash.js
const votou = {}

function verificar_eleitor(nome){
	if(votou.hasOwnProperty(nome)){
		return "Já votou"
	}else{
		votou[nome] = true
		return "Deixe votar"
	}
}

console.log(verificar_eleitor("Nil"))
console.log(verificar_eleitor("Joao"))
console.log(verificar_eleitor("Nil"))